docs/guides/getting_started.md

```diff
@@ -93,19 +93,18 @@ Did you see all that? If you did, congratulations! You're ready to go!
 
 ## Setting up on the Apify Platform
 
-Maybe you don't have Node.js installed and don't want the hassle. Or you can't install anything on your computer because you're using a company
-provided one. Or perhaps you'd just prefer to start working in the cloud right away. Well, no worries, we've got you covered.
+Maybe you don't have Node.js installed and don't want the hassle. Or you can't install anything on your computer because you're using one provided by your company. Or perhaps you'd just prefer to start working in the cloud right away. Well, no worries, we've got you covered.
 
 The [Apify platform](../guides/apify-platform) is the foundational product of
 [Apify](https://apify.com). It's a serverless cloud computing platform, specifically designed for any web automation jobs,
-that may include crawling and scraping, but really works amazing for any batch jobs and long running tasks.
+that may include crawling and scraping, but really works amazingly for any batch jobs and long-running tasks.
 
 It comes with a free account, so let's go to our [sign-up page](https://my.apify.com/sign-up) and create one, if you
 haven't already. Don't forget to verify your email. Without it, you won't be able to run any projects.
 
-Once you're in, you might be prompted by our in-app help to walk through a step-by-step guide into some of our new features. Feel free to finish that,
+Once you're in, you might be prompted by our in-app help to walk through a step-by-step guide to some of our new features. Feel free to finish that,
 if you'd like, but once you're done, click on the **Actors** tab in the left menu. To read more about **Actors**, see:
-[What is an Actor](../guides/apify-platform#what-is-an-actor).
+[What is an actor?](../guides/apify-platform#what-is-an-actor?)
```

Comment:
The `?` in the link will break it. Keep the link as is.

Reply:
I've removed the question mark from here and also the heading, so hopefully that should all work.

Just checking on workflow. I may (if I get to it) be going through other SDK docs later this week or next - would you prefer me to warn you ahead of time or just go ahead when I have the chance?

Reply:
Sorry, I meant just the second question mark. It should work like this.

```markdown
[What is an actor?](../guides/apify-platform#what-is-an-actor)
```

And yeah, feel free to edit whatever you think needs editing without telling me first. But know that documentation is not that sensitive to perfect English so I would not give it priority unless you see something truly disastrous :)

Reply:
Hey there, I only saw this now for some reason - even though I have notifications turned on it GitHub, I don't seem to reliably get them for all messages.
I don't know whether my changes were eventually merged into the getting started page in the end or whether you were waiting for me to make the change to that link. Can you check whenever you get the chance?